### PR TITLE
Adding LogR

### PR DESCRIPTION
**Pull request by davi17g**

### CODE REVIEW

File: system/system.go [hunk 1]
```diff
@@ -12,14 +12,11 @@ import (
 	"sync"
```
Comment (ashishshinde) [inline system/system.go:12]: @sud82 Is this file required?

**Comment by sud82** [comment]
Older log msgs look inconsistent. Some start with lower case and others with upper case. All should start with the lower case. We will also fix this in the operator.
